src/toolbar/toolbar-controller.js

```diff
@@ -6,11 +6,21 @@
             this.$timeout = $timeout;
             this.currentWindowService = currentWindowService;
             this.maximised = false;
+            this.summarised = false;
             currentWindowService.ready(this.onReady.bind(this));
         }
 
         onReady() {
             this.window = this.currentWindowService.getCurrentWindow();
+            this.window.getBounds(function(bounds) {
+                if (bounds.width === 230) {
```

Comment:
This `230` feels a bit like a magic number. However, the JSON file I mentioned on Slack might help with that.